templates/plugin/log_parser.conf.epp

```diff
@@ -0,0 +1,70 @@
+<%- | Optional $logfile = undef | -%>
```

Comment:
This should have a spec test that shows how the template would be rendered given the default setting. Check out an example at https://github.com/ghoneycutt/puppet-module-bind/blob/master/spec/defines/acl_spec.rb#L22-L47